tests/src/rules/prefer-default-export.js

```diff
@@ -129,4 +130,62 @@ ruleTester.run('prefer-default-export', rule, {
       }],
     }),
   ],
-})
+});
+
+context('Typescript', function() {
+  // Typescript
+  const parsers = ['typescript-eslint-parser', 'babel-eslint'];
```

Comment:
eslint 6 now requires absolute paths here
```suggestion
  const parsers = [require.resolve('typescript-eslint-parser'), require.resolve('babel-eslint')];
```